client/src/modules/shopping-list/model/selectors.js

```diff
@@ -1,6 +1,14 @@
 import _pick from 'lodash/pick';
+import _filter from 'lodash/filter';
+import _keyBy from 'lodash/keyBy';
 
 export const getShoppingList = (state, listId) =>
   _pick(state.shoppingLists.data, listId)[listId];
 export const getShoppingLists = state => state.shoppingLists.data;
+export const getLists = (state, cohortId) =>
```

Comment:
maybe this above method should be named getLists  (without 'shopping') and this one could have name like getListsForCohort? I know this is longer but we should remove 'shopping' word from code and there will have a naming conflict

Reply:
it was getListForCurrentCohort previously 😄